shared-bindings/microcontroller/Processor.c

```diff
@@ -93,9 +93,34 @@ const mp_obj_property_t mcu_processor_temperature_obj = {
     },
 };
 
+//| .. attribute:: unique id
```

Comment:
`..attribute:: uid` It should match the entry in `mcu_processor_locals_dict_table` below.